test cases/unit/12 cross prog/some_cross_tool.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python
```

Comment:
Let's make this python3 and force the use of python3 everywhere? Same for the other tool.